src/main/java/com/whichclasses/statistics/Rateable.java

```diff
@@ -0,0 +1,9 @@
+package com.whichclasses.statistics;
+
+import com.whichclasses.model.proto.TceRating;
+
+public interface Rateable {
+  public int getRatingCount(TceRating.Question question);
+  public double getAverageScore(TceRating.Question question);
+  public double getWilsonScore(TceRating.Question question);
```

Comment:
It makes sense in either context, all it requires is a collection of ratings (which can belong to an individual class or a collection of classes). You can rate a course by grabbing all class ratings and then calculating a Wilson score from that set, or you can calculate individual Wilson scores for each class and then aggregate the class results in some other way.